codalab/lib/bundle_cli.py

```diff
@@ -1561,14 +1563,26 @@ def do_run_command(self, args):
         params = {'worksheet': worksheet_uuid}
         if args.after_sort_key:
             params['after_sort_key'] = args.after_sort_key
-        new_bundle = client.create(
-            'bundles',
-            self.derive_bundle(RunBundle.BUNDLE_TYPE, args.command, targets, metadata),
-            params=params,
-        )
-
-        print(new_bundle['uuid'], file=self.stdout)
-        self.wait(client, args, new_bundle['uuid'])
+        if args.memo:
+            memoized_bundles = client.fetch(
+                'bundles',
+                params={
+                    'command': args.command,
+                    'dependencies': [uuid for uuid, target in targets],
+                },
+            )
+            if len(memoized_bundles) > 0:
```

Comment:
If there are multiple ones, we should make sure to take the latest one.

Reply:
Yes. The results are ordered by id, and we are going to return the last one if there is any.